### PR TITLE
[17.0][MIG] account_analytic_required: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 17.0

cc https://github.com/APSL 156840
@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall please review

- [Issue] https://github.com/OCA/account-analytic/issues/607

### CODE REVIEW

**Review by mpascuall — APPROVED** [review]
LGTM

**Review by lbarry-apsl — APPROVED** [review]
It works well. LGTM

**Review by leemannd — APPROVED** [review]
Base migration is good. Administrative & translations commits should be squashed

**Comment by peluko00** [comment]
> Base migration is good. Administrative & translations commits should be squashed

Done, thanks!

**Review by BernatObrador — APPROVED** [review]
LGTM, tested in runboat and code review

**Review by leemannd — APPROVED** [review]

File: account_analytic_required/tests/test_account_analytic_required.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class TestAccountAnalyticRequired(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
-        super(TestAccountAnalyticRequired, cls).setUpClass()
+        super().setUpClass()
```
Comment (carolinafernandez-tecnativa) [inline account_analytic_required/tests/test_account_analytic_required.py:12]: You should not change anything related to migration on initial commit.

**Review by carolinafernandez-tecnativa — COMMENTED** [review]
Functional review LGTM, could you please squash administrative commits?
Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for further information

File: account_analytic_required/tests/test_account_analytic_required.py (hunk 1 above)
Comment (peluko00) [reply account_analytic_required/tests/test_account_analytic_required.py:12]: I changed that in the pre-commit commit according to OCA convention and not in the migration commit

Comment (carolinafernandez-tecnativa) [reply account_analytic_required/tests/test_account_analytic_required.py:12]: If i checked OCA convention is does not say anything to change in this part:
image

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Please do not forget to squash administrative commits, thanks :)

Comment (peluko00) [reply account_analytic_required/tests/test_account_analytic_required.py:12]: > If i checked OCA convention is does not say anything to change in this part: image
> 
> https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0
> 
> Please do not forget to squash administrative commits, thanks :)

It says that ignore only pylint errors but this is a different error.
Just now i squash administrive commits

Comment (carolinafernandez-tecnativa) [reply account_analytic_required/tests/test_account_analytic_required.py:12]: I am still seeing some administrative commits.
That change according to OCA conventions as link i have sent before, it should be on migration commit. thanks!

**Review by yankinmax — APPROVED** [review]

**Comment by omalbastin** [comment]
where can we find the dependency module account_usability?

**Pull request merged**